tests/test_sql.py

```diff
@@ -107,3 +108,22 @@ def test_sql_overrides_connection_configuration(mock_conn, runner, mock_cursor):
         password="passFromTest",
         mfa_passcode=None,
     )
+
+
+def test_fail_command_when_default_config_has_too_wide_permissions(
```

Comment:
Maybe `test_main` or `test_cli` would be a better location for this test case. WDYT?

Reply:
Moved to test_main